docs/build-a-runtime/index.md

```diff
@@ -1,63 +1,87 @@
 ---
```

Comment:
Do the markdown files have to be named `index.md`? I am just wondering what it would take to split the sections into different pages.

Reply:
I called it index, so it will be the page shown at the b9/build-a-runtime/. Breaking this doc into different pages was exactly what I was thinking... It will be super easy, just start adding files.